examples/tv-app/android/java/ChannelManager.cpp

```diff
@@ -363,30 +371,273 @@ void ChannelManager::HandleGetProgramGuide(
     const chip::Optional<chip::app::DataModel::DecodableList<AdditionalInfoType>> & externalIdList,
     const chip::Optional<chip::ByteSpan> & data)
 {
-
-    // 1. Decode received parameters
-    // 2. Perform search
-    // 3. Return results
     ProgramGuideResponseType response;
-    // response.channelPagingStruct;
-    // response.programList;
-    helper.Success(response);
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    JNIEnv * env   = JniReferences::GetInstance().GetEnvForCurrentThread();
+
```

Comment:
Could you put the below here to manage the local reference
JNIEnv * env = JniReferences::GetInstance().GetEnvForCurrentThread();
VerifyOrReturn(env != nullptr, ChipLogError(Controller, "Could not get JNIEnv for current thread"));
JniLocalReferenceManager manager(env);

Reply:
fixed - thanks!